src/mavedb/lib/experiments.py

```diff
@@ -99,6 +99,17 @@ def search_experiments(
             )
         )
 
+    if search.meta_analysis is not None:
+        if not search.meta_analysis:
+            query = query.filter(
+                or_(
+                    # Keep experiments without any score sets
+                    not_(Experiment.score_sets.any()),
+                    # Keep experiments where score sets exist but have no meta_analyzes_score_sets
+                    Experiment.score_sets.any(not_(ScoreSet.meta_analyzes_score_sets.any()))
+                )
+            )
```

Comment:
I think we should also support the inverse logic here as well: If a user supplies `meta_analysis: true` we should return only meta analysis score sets.

Reply:
I agree. I'll add it.